src/rtc.c

```diff
@@ -731,6 +733,108 @@ void RTC_Alarm_IRQHandler(void)
   HAL_RTC_AlarmIRQHandler(&RtcHandle);
 }
 
+/**
+  * @brief Attach Seconds interrupt callback.
+  * @note  stm32F1 has a second interrupt capability
+  *        other MCUs map this on their WakeUp feature
+  * @param func: pointer to the callback
+  * @retval None
+  */
+void attachSecondsIrqCallback(voidCallbackPtr func)
+{
+#if defined(STM32F1xx)
```

Comment:
```suggestion
#ifdef ONESECOND_IRQn
#if defined(STM32F1xx)
```

Reply:
all the new functions are flagged with`#ifdef ONESECOND_IRQn`